hal/src/sercom/v2/i2c/pads_thumbv7em.rs

```diff
@@ -0,0 +1,143 @@
+//! Define a container for a set of SERCOM pads
+//!
+//! See the [i2c module](super) documentation for more details on declaring and
+//! instantiating a [`Pads`] type.
+
+use crate::{gpio::v2::AnyPin, sercom::v2::*, typelevel::Sealed};
+use core::marker::PhantomData;
+
+/// Container for a set of SERCOM [`Pad`]s
+///
+/// See the [module-level](crate::sercom::v2::i2c) documentation for more
+/// details on specifying a `Pads` type and creating instances.
+pub struct Pads<S, I, SDA, SCL>
+where
+    S: Sercom,
+    I: IoSet,
+    SDA: IsI2cPad<PadNum = Pad0, Sercom = S> + InIoSet<I>,
+    SCL: IsI2cPad<PadNum = Pad1, Sercom = S> + InIoSet<I>,
+{
+    sercom: PhantomData<S>,
+    ioset: PhantomData<I>,
+    sda: SDA,
+    scl: SCL,
+}
+
+impl<S, I, DI, CI> Pads<S, I, Pad<S, DI>, Pad<S, CI>>
```

Comment:
I realized that this type is exactly `PadsFromIds<S, I, DI, CI>`. Do you think it would make sense to use that alias here?